packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -7,45 +7,44 @@ category: management
 
 The Okta Application API provides operations to manage applications and/or assignments to users or groups for your organization.
 
-## Getting Started
+## Get started
 
 Explore the Apps API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/18dd817ee8abace68dd8)
 
-## Application Operations
+## Application operations
 
-### Add Application
+### Add application
 
 <ApiOperation method="post" url="/api/v1/apps" />
 
 Adds a new application to your Okta organization
 
-##### Request Parameters
+##### Request parameters
 
 
 | Parameter | Description                                                                            | Param Type | DataType                          | Required | Default |
 | --------- | -------------------------------------------------------------------------------------- | ---------- | --------------------------------- | -------- | ------- |
 | activate  | Executes [activation lifecycle](#activate-application) operation when creating the app | Query      | Boolean                           | FALSE    | TRUE    |
 | app       | App-specific name, signOnMode and settings                                             | Body       | [Application](#application-model) | TRUE     |         |
 
-##### Response Parameters
-
+##### Response parameters
 
-All responses return the created [Application](#application-model).
 
-#### Add Bookmark Application
+All responses return the created [application](#application-model).
 
+#### Add Bookmark application
 
-Adds an new bookmark application to your organization
+Adds an new Bookmark application to your organization
```

Comment:
Adds **a** new Bookmark... (not "an")

Reply:
GAH! I didn't even SEE that. LOL Thanks.